storage/notifications.go

```diff
@@ -151,6 +176,8 @@ func (b *BucketHandle) AddNotification(ctx context.Context, n *Notification) (re
 
 // Notifications returns all the Notifications configured for this bucket, as a map
 // indexed by notification ID.
+//
+// Note: Pagination is not currently supported.
```

Comment:
Let's remove this; it's not actionable for library users and they are guaranteed to receive all notifications due to entity limits.